lib/elixir/lib/calendar/iso.ex

```diff
@@ -715,6 +715,126 @@ defmodule Calendar.ISO do
     {0, 1}
   end
 
+  @doc """
+  Implements the inspect protocol for a date
+  delegated from `Date`.
```

Comment:
What do you mean by "delegated from `Date`"? I find that confusing 🙂

Reply:
Good point. I think those docs could say "Inspect implementation for `Date` in this calendar." and so on. WDYT?

Reply:
Sounds good. I would link to the `Inspect` protocol using backticks though:

> `Inspect` implementation for `Date` in this calendar.

Or:

> Implements the `Inspect` protocol for `Date` in this calendar.